src/util.js

```diff
@@ -481,6 +481,30 @@ export default {
   },
 
   /**
+   * Get native Web Cryptography api for old browsers only, including legacy
+   * implementations of the spec e.g IE11 and Safari 8/9. The default
+   * configuration is to use the api when available. But it can be deactivated
+   * with config.use_native
+   * @return {Object}   The SubtleCrypto api or 'undefined'
+   */
+  getWebCryptoPrefixed: function() {
+    if (!config.use_native) {
+      return;
+    }
+
+    var _self = this.getGlobal();
+    if (typeof _self === 'undefined') {
+      return;
+    }
+
+    if (typeof _self.crypto !== 'undefined') {
+      return _self.crypto.webkitSubtle;
```

Comment:
perfect! this is exactly as i would have changed it.

I'm not that satisfied of the name of the utility, that is more a getNonStandardWebCrypto() but it's only a matter of naming so :+1: